production/loki-mixin/dashboards/loki-deletion.libsonnet

```diff
@@ -61,15 +61,15 @@ local utils = import 'mixin-utils/utils.libsonnet';
           )
           .addPanel(
             $.newQueryPanel('Lines Deleted / Sec') +
-            g.queryPanel('sum(rate(loki_compactor_deleted_lines{' + $._config.per_cluster_label + '=~"$cluster",job=~"$namespace/%s"}[$__rate_interval])) by (user)' % compactor_matcher, '{{user}}'),
+            g.queryPanel('sum(rate(loki_compactor_deleted_lines{' + $._config.per_cluster_label + '=~"$cluster",' + compactor_matcher + '}[$__rate_interval])) by (user)', '{{user}}'),
```

Comment:
this now needs a namespace label selector since we're not selecting on `job=~"$namespace/%s"` anymore

Reply:
@cstyan this should be okay now :)